.github/workflows/build.yml

```diff
@@ -35,18 +36,20 @@ jobs:
       - name: checkout
         uses: actions/checkout@v2
       - name: lint helm chart
-        run: make lint-helm
+        uses: WyriHaximus/github-action-helm3@v2
+        with:
+          exec: helm lint ./deployment/network-operator/
   unit-tests:
-    name: Unit-tests
+    name: unit-tests
     runs-on: ubuntu-20.04
     permissions:
       contents: read
     steps:
       - name: Set up Go 1.x
         uses: actions/setup-go@v2
         with:
-          go-version: '1.15.8'
+          go-version: '1.16.6'
```

Comment:
We've got 1.15 in our go.mod, that's why I didn't use 1.16.x

Reply:
The new `Dockerfile` is using 1.16.